examples/src/routes/PivotTableDemo.jsx

```diff
@@ -27,41 +27,59 @@ export const PivotTableDemo = () => (
 
         <hr className="separator" />
 
-        <h2 id="measures-row-attributes-and-column-attributes">Example of Presorted Pivot Table</h2>
-        <ExampleWithSource for={() => <PivotTableSortingExample />} source={PivotTableSortingExampleSRC} />
-
+        <h2>Example of Presorted Pivot Table</h2>
```

Comment:
I would upper case just the first letter. This title and lots of after it seems inconsistent with other ones you have here.